src/Css/Stylesheet.php

```diff
@@ -1361,7 +1361,7 @@ private function _parse_css($str)
                                 $key = $page_selector;
 
                             default:
-                                continue;
+                                break;
```

Comment:
I think we should maybe consider that the previous incarnation of the code was written incorrectly. If we go based on the common understanding of `continue` (short circuit to the next iteration of the looping structure) then dropping out of the switch may not have been the intended operation. Even though [based on the PHP documentation](http://php.net/manual/en/control-structures.continue.php) we were indeed just `break`ing out of the switch statement.

Perhaps we should go with:

```suggestion
                                continue 2;
```